contribs/gnodev/README.md

```diff
@@ -1,30 +1,67 @@
+by providing immediate feedback on code changes.
 
-We will only give a quick overview below. You may find the official documentation at [docs/gno-tooling/gnodev.md](../../docs/gno-tooling/cli/gnodev.md).
+Please note that this is a quick overview. For a more detailed guide, refer to the official documentation at
+[docs/gno-tooling/gnodev.md](../../docs/gno-tooling/cli/gnodev.md).
 
 ### Synopsis
-**gnodev** [**-minimal**] [**-no-watch**] [**PKG_PATH ...**]
+**gnodev** [**options**] [**PKG_PATH ...**]
 
 ### Features
-- **In-Memory Node**: Gnodev starts an in-memory node, and automatically loads
-  the **examples** folder and any user-specified paths.
-- **Web Interface Server**: Starts a `gnoweb` server on `localhost:8888`.
-- **Hot Reload**: Monitors the example packages folder and specified directories for file changes,
-  reloading the package and automatically restarting the node as needed.
-- **State Maintenance**: Ensures the current state is preserved by replaying all transactions.
+-  **In-Memory Node**: Gnodev starts an in-memory node, automatically loading the **examples** folder and any
+   user-specified paths.
+-  **Web Interface Server**: Gnodev starts a `gnoweb` server on [`localhost:8888`](https://localhost:8888).
+-  **Balances and Keybase Customization**: Set account balances, load them from a file, or add new accounts via a flag.
+-  **Hot Reload**: Monitors the **examples** folder and specified directories for file changes, reloading the
+   package and automatically restarting the node as needed.
+-  **State Maintenance**: Ensures the previous node state is preserved by replaying all transactions.
+-  **Transaction Manipulation**: Allows for interactive cancellation and redoing of transactions.
+-  **State Export**: Export the current state at any time in a genesis doc format.
 
 ### Commands
-While `gnodev` is running, the user can trigger specific actions by pressing
-the following combinations:
-- **H**: Display help information.
-- **R**: Reload the node, without resetting the state.
-- **Ctrl+R**: Reset the current node state.
-- **Ctrl+C**: Exit `gnodev`.
+While `gnodev` is running, trigger specific actions by pressing the following combinations:
+-  **H**: Display help information.
+-  **A**: Display account balances.
+-  **R**: Reload the node manually.
+-  **P**: Cancel the last action.
+-  **N**: Redo the last cancelled action.
+-  **Ctrl+S**: Save the current state.
+-  **Ctrl+R**: Restore the saved state.
+-  **E**: Export the current state to a genesis file.
+-  **Cmd+R**: Reset the current node state.
+-  **Cmd+C**: Exit `gnodev`.
+
+### Usage
+Run `gnodev` followed by any specific options and/or package paths. The **examples** directory is loaded
+automatically. Use `--minimal` to prevent this.
+
+Example:
+```
+gnodev --add-account <bech32/name1>[:<amount1>] ./myrealm
+```
+
+### `gnobro`: realm interface
```

Comment:
_minor_ should not be this title at the same level of `gnodev` ? (##)

Reply:
For now, `gnobro` is a subtool of gnodev as it lives inside the gnodev repo